packages/datadog-plugin-pg/src/index.js

```diff
@@ -46,21 +46,46 @@ function createWrapQuery (tracer, config) {
         })
       }
 
-      pgQuery.callback = scope.bind((err, res) => {
-        if (err) {
-          span.addTags({
-            'error.type': err.name,
-            'error.msg': err.message,
-            'error.stack': err.stack
-          })
-        }
+      if (originalCallback) {
+        pgQuery.callback = scope.bind((err, res) => {
+          if (err) {
+            span.addTags({
+              'error.type': err.name,
+              'error.msg': err.message,
+              'error.stack': err.stack
+            })
+          }
 
-        span.finish()
+          span.finish()
```

Comment:
This and the previous statement could be moved to a function with an optional error parameter. For example:

```js
function finish (span, error) {
  span.setTag('error', error)
  span.finish()
}
```